src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -286,6 +269,33 @@ private void createCommitStatus(Run<?, ?> build,
         }
     }
 
+    public void createCommitStatus(Job<?, ?> project,
```

Comment:
Ah I see the problem. So basically you cannot call this method.

Should this method be extracted and moved to a common place or is there no abstraction level which makes sense for both callers?

I personally feel its a bit edgy that the interface now has the `createCommitStatus` method.

Reply:
You're right, it is out of place there the way it is. I wanted to make an easy implementation of `GhprbPullRequest.createCommitStatus(GHCommitState state, String message)` for future developers, but maybe that's not necessary.
I could create `GhprbCommitStatus.onBuildSkipped` that would make more sense for `GhprbCommitStatus` being an event interface. But also, the only other implementor is `GhprbNoCommitStatus` and completely unused. I could remove the interface, remove `GhprbNoCommitStatus`, and then it doesn't seem as edgy.

I thought that level of refactor might be out of scope, but I'm open to it. Of these options I am in favor of removing the unused classes & abstraction. Which direction would you prefer?

Reply:
Sorry for the late response. I think the best option is to remove the unused classes & abstraction.

Do you have an Idea in your head how this could look like in the end?

Reply:
Just pushed it. It was relatively concise and simple. It exposed to me that there's another class `GhprbSimpleStatusDescriptor` which is completely unused, but I left that as it is not in scope of this change.